backend/de.metas.business/src/main/java/de/metas/handlingunits/impl/CreateShipperTransportationRequest.java

```diff
@@ -52,4 +52,6 @@ public class CreateShipperTransportationRequest
 	 * Should be {@code true} if the user is supposed to physically in the real world find and put into the package exactly those HUs that metasfresh picked on the fly.
 	 */
 	boolean assignAnonymouslyPickedHUs;
+
+	boolean isSOTrx;
```

Comment:
loud thinking: should we set it as default true? or shall we use SOTrx obj?

Reply:
I thought about defaulting it, but it's only a handful of usages, and are fairly straight-forward. Also, those usages are 50-50 in the Sales/Purchase usages, so decided not to default.
Changed to use SOTrx.